src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -1026,6 +1026,28 @@ int main(int argc, char** argv)
   using IndexSet = slam::PositionSet<>;
   using Circle = primal::Sphere<double, DIM>;
 
+#if defined(AXOM_USE_UMPIRE)
+  //---------------------------------------------------------------------------
+  // Memory resource.  For testing, choose device memory if appropriate.
+  //---------------------------------------------------------------------------
+  const std::string umpireResourceName =
+    params.policy == RuntimePolicy::seq || params.policy == RuntimePolicy::omp
+    ? "HOST"
+    :
+  #if defined(UMPIRE_ENABLE_DEVICE)
+    "DEVICE"
+  #elif defined(UMPIRE_ENABLE_UM)
+    "UM"
+  #elif defined(UMPIRE_ENABLE_PINNED)
+    "PINNED"
+  #else
+    "HOST"
+  #endif
+    ;
+  auto& rm = umpire::ResourceManager::getInstance();
+  umpire::Allocator ummpireAllocator = rm.getAllocator(umpireResourceName);
```

Comment:
please adjust spelling of `ummpireAllocator`

Reply:
OK.  Thanks for catching.